src/languages/scala.js

```diff
@@ -107,11 +107,22 @@ export default function(hljs) {
   const METHOD = {
     className: 'function',
     beginKeywords: 'def',
-    end: /[:={\[(\n;]/,
+    end: /(?=[:={\[(\n;])/,
     excludeEnd: true,
     contains: [ NAME ]
   };
 
+  const USING_PARAM_CLAUSE = {
+    begin: [
+      /[(]\s*/, // Opening `(` of a parameter or argument list
```

Comment:
```suggestion
      /\(\s*/, // Opening `(` of a parameter or argument list
```

Prefer escape vs character group.

Reply:
Done